ripc-protocol-tcp/src/main/java/io/ripc/protocol/tcp/TcpHandler.java

```diff
@@ -0,0 +1,8 @@
+package io.ripc.protocol.tcp;
+
+import org.reactivestreams.Publisher;
+
+public interface TcpHandler<R, W> {
```

Comment:
As a general rule, I'm not a fan of duplicating names in both packages and class names. It also doesn't "handle" `Tcp`, it handles a `Connection`. If needing a qualifier to `Handler`, it should probably be `ConnectionHandler`.